### ISSUE TITLE
Kytos installation from wheel doesn't install config files

### ISSUE THREAD

**Issue by gleybersonandrade** [kytos/kytos#922]
When I try to install kytos by Pypi, it doesn't work for 2019.1b2 and 2019.1b3. This problem occurs because the 'etc' and 'var' folders aren't be created in the environment.

Steps to reproduce:

```
python3.6 -m venv env
source env/bin/activate
pip install --upgrade pip setuptools wheel
```
and
```
pip install kytos==2019.1b2
```
or
```
pip install kytos==2019.1b3
```

**note:** kytos 2019.1b2 returns an dependency error and kytos 2019.1b3 returns a notification: _There is no config file_.

**Comment by hdiogenes** [comment]
It seems that this is a recurrent problem with binary packages in Python, and the most widely accepted solution to this problem is to change the time when those files are created ([see this answer in SO](https://stackoverflow.com/questions/47460804/copy-configuration-file-on-installation/47465374#47465374)). Instead of creating them at installation time, create them at "bootstrap" time: check for the files every time kytos is run and create them from the templates if they're not present.

**Comment by hdiogenes** [comment]
Note: this is an old issue (#471) in Kytos, but as the discussion there was huge, I've closed the old thread and linked it here for reference.

**Comment by hdiogenes** [comment]
To create the configuration files from a wheel we'd have to do it on runtime instead of install time. For now the workaround is that we're not distributing wheel files, only the source distribution.

**Comment by hdiogenes** [comment]
Fixed in #1042 / #1053.
